ecosystem/platform/server/lib/logging/logs.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+module Logging
+  module Logs
+    REQUEST_ID_KEY = 'request_id'
+    USER_ID_KEY = 'user_id'
+
+    def log(message_or_object, message = nil)
+      result = []
+      result << "time=#{Time.now.to_i}"
+      result << "class=#{self.class}"
+
+      request_id = Thread.current.thread_variable_get(REQUEST_ID_KEY)
+      result << "request_id=#{request_id}"
+
+      user_id = Thread.current.thread_variable_get(USER_ID_KEY)
+      result << "user_id=#{user_id}" if user_id.present?
+
+      if message.nil?
+        message = message_or_object
+      else
+        object = message_or_object
+        result << "object_class=#{object.class}"
+        result << "object_id=#{object.id}" if object.respond_to?(:id)
+      end
+
+      message = message.gsub('"', '\"')
+      result << "message=\"#{message}\""
```

Comment:
quick q; why this vs json, and what happens if `message` has a `something=` in it?

Reply:
I did it this way because lograge outputs key=value by default, but I like the idea of outputting JSON better. Will update.